contrib/gocql/gocql/gocql_test.go

```diff
@@ -30,7 +30,10 @@ func newCassandraCluster() *gocql.ClusterConfig {
 // TestMain sets up the Keyspace and table if they do not exist
 func TestMain(m *testing.M) {
 	cluster := newCassandraCluster()
-	session, _ := cluster.CreateSession()
+	session, err := cluster.CreateSession()
+	if err != nil {
+		t.Fatalf("skipping test: %v\n", err)
```

Comment:
Good with this approach, but we may need to update our waiting script so that we wait for Cassandra to be up and running: https://github.com/DataDog/dd-trace-go/blob/f40ac9c6093ac2921b4fce21218dbd220b2ee1a2/wait-for-services.sh